metadata.json

```diff
@@ -49,15 +47,13 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "16.04",
```

Comment:
I think Ubuntu 16.04 is still supported and works? In that case I think we can keep it?

Reply:
ack will add back